core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/jmx/camel/CamelFacadeSupport.java

```diff
@@ -73,19 +71,18 @@ protected Object newProxyInstance(ObjectName objectName, Class<?> interfaceClass
      * @param connection not <code>null</code>
      * @param managementName to find a specific context by its management name
      * @return Set with ObjectName-elements
-     * @throws MalformedObjectNameException 
-     * @throws IOException 
      */
-    protected Set<ObjectName> findCamelContexts(MBeanServerConnection connection, String managementName) throws MalformedObjectNameException, IOException {
+    protected Set<ObjectName> findCamelContexts(MBeanServerConnection connection, String managementName) throws Exception {
         String id = managementName != null ? managementName : camelContextManagementName;
 
         ObjectName name;
         if (id != null) {
-            name = new ObjectName(ORG_APACHE_CAMEL_CONTEXT + managementName + ",type=context,*");
+            name = new ObjectName("org.apache.camel:context=" + managementName + ",type=context,*");
```

Comment:
it looks alike that you have reverted some commits, this one https://github.com/fusesource/fuseide/commit/61bbc27bd26779ce718be75458909ec3b2c8e5ba for this part